models/enterprise_sonic/interfaces/replaced_example_01.txt

```diff
@@ -5,25 +5,36 @@
 #
  show interface status | no-more
```

Comment:
Please modify this example to show that "replaced" state removes all previous configuration for the interfaces specified in the playbook and installs on those interfaces only the configuration specified in the playbook. It is important here to include at least one or two examples of configuration that is removed from the "target" interfaces because it is not specified in the playbook. (For example, show that an interface that is "enabled" before playbook is "disabled" if the playbook specifies configuration for that interface without explicitly including the "enabled" state.)

Reply:
Thanks for the updates.

This issue can be marked "Resolved".